TeamCode/src/main/java/org/firstinspires/ftc/teamcode/CougarRobot.java

```diff
@@ -0,0 +1,259 @@
+package org.firstinspires.ftc.teamcode;
```

Comment:
@jadonx I would suggest using a different name for this file, `CougarRobot` doesn't tell me what is different about this file vs others - is this for a field-centric OpMode? Autonomous?